### ISSUE TITLE
JS npm starter fails to install

### ISSUE THREAD

**Issue by edmulraney** [pmlopes/vertx-starter#62]
Box: Windows 7

Steps:
- Use https://vertx-starter.jetdrone.xyz/#npm to generate empty project
- Run `npm install` in project

Received:
```
$ npm i

> vertx_test@0.0.1 postinstall C:\dev\code\vertx_test
> es4x install

Exception in thread "main" java.lang.ClassCastException: java.util.LinkedHashMap cannot be cast to java.util.List
        at io.reactiverse.es4x.commands.InstallCommand.processPackageJson(InstallCommand.java:81)
        at io.reactiverse.es4x.commands.InstallCommand.installNodeModules(InstallCommand.java:217)
        at io.reactiverse.es4x.commands.InstallCommand.run(InstallCommand.java:128)
        at io.vertx.core.impl.launcher.VertxCommandLauncher.execute(VertxCommandLauncher.java:226)
        at io.vertx.core.impl.launcher.VertxCommandLauncher.dispatch(VertxCommandLauncher.java:361)
        at io.vertx.core.impl.launcher.VertxCommandLauncher.dispatch(VertxCommandLauncher.java:324)
        at io.reactiverse.es4x.ES4X.main(ES4X.java:55)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vertx_test@0.0.1 postinstall: `es4x install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vertx_test@0.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Dev\devtools\cache\npm\_logs\2019-06-18T16_48_17_576Z-debug.log
```

**Comment by okou19900722** [comment]
it seems need a array, but the template of package.json give a object

https://github.com/reactiverse/es4x/blob/c9a7196eaa79cc56cc0f1c3dee7ba6eb89610913/pm/src/main/java/io/reactiverse/es4x/commands/InstallCommand.java#L80

https://github.com/pmlopes/vertx-starter/blob/60875eca01b21157dc09550fc44bbef0b5e417a9/templates/npm/package.json#L42-L46

I think it must be 
```json
{
    "mvnDependencies":[
        "{{ groupId }}:{{ artifactId }}:{{ version}}......."
    ]
}
```

**Comment by pmlopes** [comment]
Ah yes the format changed and I haven't updated the template...

**Comment by pmlopes** [comment]
Should be fixed by: e17b28381825344771089f0e8385f5a7bd43261d

Build and deploy in progress give it a few minutes and clear your cache and it should be fixed!
